modules/compose/compose_api.go

```diff
@@ -32,14 +32,6 @@ func (f stackUpOptionFunc) applyToStackUp(o *stackUpOptions) {
 	f(o)
 }
 
-//nolint:unused
-type stackDownOptionFunc func(do *api.DownOptions)
-
-//nolint:unused
-func (f stackDownOptionFunc) applyToStackDown(do *api.DownOptions) {
-	f(do)
-}
-
```

Comment:
In line L561, I'd also add the testcontainers labels to all the created volumes, like this:

```go
	for key, n := range proj.Volumes {
		n.Labels = map[string]string{
			api.ProjectLabel: proj.Name,
			api.VolumeLabel:  n.Name,
			api.VersionLabel: api.ComposeVersion,
		}

		for k, label := range testcontainers.GenericLabels() {
			n.Labels[k] = label
		}

		proj.Volumes[key] = n
	}
```

Reply:
I'll do that in a follow up PR if you don't mind as there's some pending changes to how that works which will make it easier.

Reply:
SGTM, thanks